packages/@uppy/companion/src/server/provider/zoom/adapter.js

```diff
@@ -127,3 +127,7 @@ exports.getSize = (item) => {
   }
   return item.total_size
 }
+
+exports.getItemTopic = (item) => {
+  return item.topic
```

Comment:
Is it always available, or should we check for undefined?

Reply:
it should always be available. We tack it onto the recording files on line 230 in the provider/zoom/index.js file with `.map(item => { return { ...item, topic: results.topic } })` so that meetings and their child recording files will both have topics